pennylane/ops/op_math/controlled_ops.py

```diff
@@ -119,22 +120,61 @@ def _unflatten(cls, data, metadata):
             data[0], control_wires=metadata[0], control_values=metadata[1], work_wires=metadata[2]
         )
 
+    # pylint: disable=arguments-differ, too-many-arguments, unused-argument
+    @classmethod
+    def _primitive_bind_call(
+        cls,
+        base,
+        control_wires: WiresLike,
+        wires: WiresLike = (),
+        control_values=None,
+        unitary_check=False,
+        work_wires: WiresLike = (),
+    ):
+        wires = Wires(()) if wires is None else Wires(wires)
+        work_wires = Wires(()) if work_wires is None else Wires(work_wires)
+
+        if getattr(base, "wires", False) and len(wires) != 0:
+            warnings.warn(
+                "base operator already has wires; values specified through wires kwarg will be ignored."
+            )
+            wires = Wires(())
```

Comment:
```suggestion
        if hasattr(base, "wires", False) and len(wires) != 0:
            warnings.warn(
                "base operator already has wires; values specified through wires kwarg will be ignored."
            )
            wires = Wires(())
```

What if `base.wires` is empty? Then, `getattr`'s output would be falsey and this can be a problematic check. This is not possible with `QubitUnitary` so it's a non-issue, but just suggesting this for extra bullet-proofing and for future reference.